src/lib.rs

```diff
@@ -117,10 +121,13 @@ pub mod pallet {
 			+ Ord
 			+ Default
 			+ Debug
-			+ Into<AccountId32>;
+			+ Into<AccountId32>
+			+ From<AccountId32>;
```

Comment:
why did this change? I think the `Into` trait bound is redundant

From<AccountId32> allows conversion from AccountId32 to RelayChainAccountId, and also conversion from RelayChainAccountId into AccountId32 because [From<T> for U implies Into<U> for T](https://doc.rust-lang.org/std/convert/trait.From.html)

Into<AccountId32> allows conversion from RelayChainAccountId to AccountId32

Reply:
If I remove the Into and leave only the from the pallet does not compile with the error:

the trait `From<<T as pallet::Config>::RelayChainAccountId>` is not implemented for `AccountId32`